pilot/pkg/proxy/envoy/v2/xds_local_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2018 Istio Authors
```

Comment:
I would move this to /tests/util (top level)

Reply:
will do.